### ISSUE TITLE
VIdeo and Audio RTC button not available in Video assignment feedback

### ISSUE THREAD

**Issue by cdipe** [kaltura/moodle_plugin#274]
Moodle version 3.6.6
Kaltura plugin 4.2.4
Moodle has its own video- and audio recorder built into the Atto editor, and also for Tiny MCE, called Video and Audio RTC. This is for quick lokal recording of short media, for example as feedback, as a regular file.
We have this activated for both the editors, and it is working everywhere in Moodle except for in the feedback in Kaltura Video Assignment, where the RTC buttons are not available in any of the editors. I cannot find any differences in the permissions settings.

**Comment by muli** [comment]
Hi @cdipe,

Seems like this behavior is intentional: https://github.com/kaltura/moodle_plugin/blame/4545c702bebd52315f3ec9ab247638d3a6b1a205/mod/kalvidassign/single_submission_form.php#L203.

Note that this is very old code, but I guess that the reason behind this limitation is either:
1. We want users to use our tools.
2. It didn't work correctly.

Looping in @idodido to decide whether he wants to keep this behavior or change it.

**Comment by cdipe** [comment]
Thank you, @muli! I can see the point in wanting the users to use Kaltura, but to them it becomes just an unexpected behaviour / error since the function exists everywhere else in the Moodle text editors. Mostly they use the Kaltura video, but for short feedbacks they find this tool much quicker.

**Comment by muli** [comment]
@cdipe, I agree with you, I was just trying to come up with reasons why this limitation makes sense and it is my speculation only. :)

Anyway, let's wait for Ido's decision.
